src/functions/sendslack/schema.ts

```diff
@@ -0,0 +1,81 @@
+export default {
```

Comment:
I think that the timebank has API spec that can be used to generate all types that you needs. You can ask me more about this.